src/azure-cli/azure/cli/command_modules/storage/tests/hybrid_2020_09_01/test_storage_account_scenarios.py

```diff
@@ -0,0 +1,248 @@
+    @ResourceGroupPreparer(location='southcentralus')
+    def test_storage_create_default_sku(self, resource_group):
+        name = self.create_random_name(prefix='cli', length=24)
+        create_cmd = 'az storage account create -n {} -g {} --https-only '.format(name, resource_group)
+        self.cmd(create_cmd, checks=[JMESPathCheck('sku.name', 'Standard_RAGRS')])
+
+    def test_show_usage(self):
+        self.cmd('storage account show-usage -l westus', checks=JMESPathCheck('name.value', 'StorageAccounts'))
+
+    # @ResourceGroupPreparer()
+    # @StorageAccountPreparer()
+    # def test_logging_operations(self, resource_group, storage_account):
+    #     connection_string = self.cmd(
+    #         'storage account show-connection-string -g {} -n {} -otsv'.format(resource_group, storage_account)).output
+
+    #     self.cmd('storage logging show --connection-string {}'.format(connection_string), checks=[
+    #         JMESPathCheck('blob.read', False),
+    #         JMESPathCheck('blob.retentionPolicy.enabled', False)
+    #     ])
+
+    #     self.cmd('storage logging update --services b --log r --retention 1 '
+    #              '--service b --connection-string {}'.format(connection_string))
+
+    #     self.cmd('storage logging show --connection-string {}'.format(connection_string), checks=[
+    #         JMESPathCheck('blob.read', True),
+    #         JMESPathCheck('blob.retentionPolicy.enabled', True),
+    #         JMESPathCheck('blob.retentionPolicy.days', 1)
+    #     ])
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_metrics_operations(self, resource_group, storage_account_info):
+        self.storage_cmd('storage metrics show', storage_account_info) \
+            .assert_with_checks(JMESPathCheck('file.hour.enabled', True),
+                                JMESPathCheck('file.minute.enabled', False))
+
+        self.storage_cmd('storage metrics update --services f --api true --hour true --minute true --retention 1 ',
+                         storage_account_info)
+
+        self.storage_cmd('storage metrics show', storage_account_info).assert_with_checks(
+            JMESPathCheck('file.hour.enabled', True),
+            JMESPathCheck('file.minute.enabled', True))
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(parameter_name='account_1')
+    @StorageAccountPreparer(parameter_name='account_2')
+    def test_list_storage_accounts(self, account_1, account_2):
+        accounts_list = self.cmd('az storage account list').get_output_in_json()
+        assert len(accounts_list) >= 2
+        assert next(acc for acc in accounts_list if acc['name'] == account_1)
+        assert next(acc for acc in accounts_list if acc['name'] == account_2)
+
+    # @ResourceGroupPreparer()
+    # @StorageAccountPreparer()
+    # def test_renew_account_key(self, resource_group, storage_account):
```

Comment:
it could work with ADDS configured.
If it is supported in Azure Stack, we need to work on reverting the test.